osh/word_eval.py

```diff
@@ -1488,7 +1666,7 @@ def EvalWordSequence2(self, words, allow_assign=False):
       argv: list of string arguments, or None if there was an eval error
     """
     if self.exec_opts.simple_word_eval:
-      return self.StaticEvalWordSequence2(words, allow_assign)
+      return self.StaticEvalWordSequence2(words, allow_assign) # follow
```

Comment:
What is this comment for?

Also I use 2 spaces before #, although I'm won't  block on anything on that until we have an autoformatter...

Reply:
Whoops!  That was just me trying to keep track while manually tracing through some calls.  I removed it (actually in 2 places): 37dcecc8c886c3ee8c0841d0cea86348e46dd5ff

Reply:
For any future work I can jump right to using tagswitch.  Do you prefer that?